<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>180</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Settings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="hostLabel">
                <property name="text">
                 <string>Host Name:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="hostLE">
                <property name="text">
                 <string>localhost</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="portL">
                <property name="text">
                 <string>port:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="portLE">
                <property name="text">
                 <string>3050</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="userL">
                <property name="text">
                 <string>username:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="userLE">
                <property name="text">
                 <string>sysdba</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="passwordL">
                <property name="text">
                 <string>passwodr:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="passwordLE">
                <property name="text">
                 <string>masterkey</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="dbfileL">
              <property name="text">
               <string>database file</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="dbFileLE"/>
            </item>
            <item>
             <widget class="QPushButton" name="dbFilePB">
              <property name="text">
               <string>Wybierz</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="outQueryL">
              <property name="text">
               <string>output query file</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="outQueryLE"/>
            </item>
            <item>
             <widget class="QPushButton" name="outQueryPB">
              <property name="text">
               <string>Wybierz</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Settings 2</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Exclude tables:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="tablesCB"/>
          </item>
          <item>
           <widget class="QToolButton" name="addTB">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="removeTB">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="tablesLV">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="connectPB">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Połącz</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="searchTextLE"/>
        </item>
        <item>
         <widget class="QPushButton" name="searchPB">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Szukaj</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Zapytanie sql: query</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="queryLE"/>
          </item>
          <item>
           <widget class="QPushButton" name="queryExecPB">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Wykonaj</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QListWidget" name="searchLW"/>
    </item>
    <item row="4" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QProgressBar" name="tablesPB">
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="format">
           <string>%v/%m</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="tableProgressL">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QProgressBar" name="columnsPB">
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="format">
           <string>%v/%m</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="columnProgressL">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
